site/content/docs/dashboards/_index.md

```diff
@@ -24,18 +24,16 @@ An example of a dashboard is [status.checklyhq.com](https://status.checklyhq.com
 
 ## Adding checks to your dashboard
 
-In order to show specific checks in a dashboard, you will need to do this:
+By default, if no check is specified, all checks in the account will be shown in the dashboard. If instead you want to show show specific checks, this is how it's accomplished:
 
 1. First add tags to one or more checks you would like to show. This is done by editing the check(s). 
 
 ![Tags](/docs/images/dashboards/tags.png)
 
-Don't forget to press Enter to save the tag before you save your updated check!
+You'll need to press Enter to save the tag before you save your updated check!
```

Comment:
I think the 1. & 2. points mess up the page a bit with the indentation (it looks more confusing than it should). My 2 cents.